mailpoet/webpack.config.js

```diff
@@ -65,7 +66,7 @@ const baseConfig = {
       asyncqueue: 'vendor/jquery.asyncqueue.js',
     },
   },
-  plugins: [],
+  plugins: [new ForkTsCheckerWebpackPlugin()],
```

Comment:
When I was working and adding my changes it felt like doing things blindly, yes mostly TS is covered by most code editors/IDEs but I've seen them going crazy with TS as well even the best ones like PHPStorm/WebStorm/VSCode, so added this and felt it to be very fast for our project since it's pretty small, I'm expecting the checks to remain lightweight for some time to come unless we end up with a massive TS codebase.

1) I'd prefer not leaving the benefits of TS to be guarded but developers' IDE/editors
2) The hurdle it has on CPU is pretty minimal compared the benefit it provides ( ~instant check and feedback on developer mistakes when using TS )
3) This is one of those topics that we can easily give it a try and change/remove it in future if we find it to be redundant or cumbersome.
These said I'm fine with breaking this to the Slack and asking for others' ideas on this, but treating it as experimental would do as well.

P.S: I'll update the lock file in the same commit.

Reply:
@samnajian Yeah, I guess I agree with "let's try to use it" and if it ever gets too heavy-weight we can revisit.

P.S.: Feel free to take all the "commit"-related comments just as an idea for next time. I'll leave up to you what you want/don't want to fix.